pkg/controllers/repositoryserver/repositoryserver_controller_test.go

```diff
@@ -46,7 +46,6 @@ import (
 	"github.com/kanisterio/kanister/pkg/poll"
 	"github.com/kanisterio/kanister/pkg/resource"
 	"github.com/kanisterio/kanister/pkg/secrets"
-	"github.com/kanisterio/kanister/pkg/secrets/repositoryserver"
```

Comment:
it seems that the import without an alias would be preferred, even if it results in a larger change
@hairyhum @pavannd1 @viveksinghggits thoughts?

Reply:
ok

Reply:
Yes, let's use import without alias.